internals/daemon/response.go

```diff
@@ -169,22 +150,26 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+func Reponsef(status int, format string, v ...interface{}) Response {
+	res := &client.Error{}
+	if len(v) == 0 {
+		res.Message = format
+	} else {
+		res.Message = fmt.Sprintf(format, v...)
+	}
```

Comment:
```suggestion
	res.Message = fmt.Sprintf(format, v...)
```

Reply:
> res.Message = fmt.Sprintf(format, v...)

No, there's an actual difference here. Try this:  `Responsef(404, "-%- funny external error -%-")`.

Please feel free to document the difference, so the next person doesn't feel tempted.

Reply:
This is existing logic in existing code, right?

Reply:
As per Gustavo's comment, I will document the detail here: We do have an -'f' type mechanism, but with a special exist clause that if no arguments are supplied, formatting is disabled. In this sense, I think its OK for now to proceed with the ```Responsef``` naming. I will document the details.

Reply:
Just to note, my point was indeed that an `f` function should behave consistently and always format even if there's no format argument.

Reply:
Documented. Resolving.